app/api/auth.py

```diff
@@ -147,9 +148,30 @@ def login():
                     
             role_response = jwt.decode(token['data']['token'], None, False)
             print(role_response)
-            print('role_response')
-
             if role_response["role"] == "Admin":
+                payload_all_disabled_users_details = {"action": "show_disabled_users", "token": token['data']['token']}
```

Comment:
move this to a scheduler which works once a day or once a week.

admin login already takes so long this api call with increase time further